### PR TITLE
MAT-721 FHIR attributes should be loaded only if FHIR_EDIT is enabled.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/mat/server/CQLConstantServiceImpl.java [hunk 1]
```diff
@@ -175,6 +161,35 @@ public CQLConstantContainer getAllCQLConstants() {
         return cqlConstantContainer;
     }
 
+    private boolean isFhirEditEnabled() {
+        return featureFlagService.findFeatureFlags().getOrDefault(FeatureFlagConstant.FHIR_EDIT, false);
+    }
```
Comment (jkotanchik-SB) [inline src/main/java/mat/server/CQLConstantServiceImpl.java]: Would it make sense to move these kind of checks into the FeatureFlagService? Perhaps, `featureFlagService.isFhirEditEnabled()`.

Comment (serhii-ilin) [reply src/main/java/mat/server/CQLConstantServiceImpl.java]: Well, maybe you're right.
Need to look into that.

**Review by jkotanchik-SB — APPROVED** [review]

**Review by jkotanchik-SB — APPROVED** [review]
Looks good.

**Review by AndrewBird81 — APPROVED** [review]
LGTM

File: src/main/java/mat/server/CQLConstantServiceImpl.java [hunk 2]
```diff
@@ -175,6 +157,31 @@ public CQLConstantContainer getAllCQLConstants() {
...
+            if (!fhirElement.isEmpty()) {
+                fhirAttributeSet.add(fhirElement);
+            }
+            if (fhirElement.isEmpty() || fhirResource.isEmpty()) {
+                continue;
+            }
+            String fhirResourceId = hashForId(fhirResource);
+            String fhirElementId = hashForId(fhirResource + "--" + fhirElement);
+            FhirDataType fhirDataType =
+                    cqlConstantContainer.getFhirDataTypes().computeIfAbsent(fhirResource, s -> new FhirDataType(fhirResourceId, fhirResource));
```
Comment (cday-semanticbits) [inline src/main/java/mat/server/CQLConstantServiceImpl.java]: nice

**Review by cday-semanticbits — APPROVED** [review]
Looks good to me.

**Pull request merged**